core/src/test/java/org/frankframework/pipes/DelayPipeTest.java

```diff
@@ -27,7 +27,7 @@ public void testUnInterruptedSession() throws Exception {
 		pipe.configure();
 		pipe.start();
 		PipeRunResult prr = doPipe(pipe, input, session);
-		String result = Message.asString(prr.getResult());
+		String result = prr.getResult().asString();
```

Comment:
This was the most used variant of asString - with the 'prr.getResult()'

Reply:
Vroeger was het result een object :) vandaar.

Reply:
Het enige is dat er nu wel eens een NPE zou kunnen komen.

Maar als dat gebeurt, dan is er al iets misgegaan in de test dus dat is ook niet zo erg.